mdanalysissandbox/bruce.py

```diff
@@ -0,0 +1,19 @@
+""" this is my code module
+"""
+
+import numpy as np
+
+def cound_names(ag, name):
+	"""count how many atoms have a particular name
+
+	parameters
```

Comment:
Capitalize "Parameters".

Reply:
Sorry, got this wrong.

- need Parameters section that documents input
- need Returns section as you have

See https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#writing-docstrings